core/dbt/contracts/graph/parsed.py

```diff
@@ -211,6 +211,20 @@ def write_node(self, target_path: str, subdirectory: str, payload: str):
         write_file(full_path, payload)
         return full_path
 
+    @property
+    def node_info(self):
+        node_info = {
+            "node_path": self.path,
+            "node_name": self.name,
+            "unique_id": self.unique_id,
+            "resource_type": self.resource_type.value,
+            "materialized": self.config.get('materialized'),
+            "node_status": str(self._event_status.get('node_status')),
+            "node_started_at": self._event_status.get("started_at"),
+            "node_finished_at": self._event_status.get("finished_at")
+        }
+        return node_info
```

Comment:
Does this mean we have multiple definitions for the node info structure? why do we need this one?

Reply:
Yeah. I made an attempt to put it in a separate class to be included but dataclasses makes the order of default and non-default attributes irritating.  It's certainly better to only have one though, so I tweaked it and made it work.